src/Files.App/Actions/Navigation/CloseActivePaneAction.cs

```diff
@@ -20,23 +20,20 @@ public RichGlyph Glyph
 			=> new("\uE89F");
 
 		public bool IsExecutable
-			=> context.IsMultiPaneActive;
+			=> ContentPageContext.IsMultiPaneActive;
 
-		public ClosePaneAction()
+		public CloseActivePaneAction()
 		{
-			context = Ioc.Default.GetRequiredService<IContentPageContext>();
-
-			context.PropertyChanged += Context_PropertyChanged;
+			ContentPageContext.PropertyChanged += ContentPageContext_PropertyChanged;
 		}
 
 		public Task ExecuteAsync(object? parameter = null)
 		{
-			context.ShellPage!.PaneHolder.CloseSecondaryPane();
-
+			ContentPageContext.ShellPage!.PaneHolder.CloseActivePane();
```

Comment:
```suggestion
			ContentPageContext.ShellPage?.PaneHolder.CloseActivePane();
```
It should be null safe to avoid crashes, even if IsExecutable has a null check.